src/cli/hook-utils/get-check-update-data.js

```diff
@@ -6,6 +6,15 @@ const exec = util.promisify(require('child_process').exec);
 const SLACK_JSON_SDKS = ['@slack/bolt', '@slack/deno-slack-sdk'];
 let dependencyExports = {};
 
+/**
+ * Wraps and parses the output of the exec() function
+ * @param command the command being run by the exec() function
+ */
+async function execWrapper (command) {
+  const { stdout } = await exec(command);
+  return stdout.replace(/\n$/, '');
```

Comment:
Can also use `trim()` to trim both leading and trailing whitespace.
```suggestion
  return stdout.trim();
```